AutomatedTesting/Passes/PassTemplates.azasset

```diff
@@ -0,0 +1,561 @@
+{
```

Comment:
I'm guessing this used to just use the one from the Atom gem, but now that we've added a custom pass we need this file. Is there any way we could only specify the new template and fall back to the atom gem for the rest? I thought we had some kind of include or import functionality in json now, but I'm not sure.

Reply:
Yes. Talked to Vicky and I got a solution to reduce the redundancy and move it to the gem. But it requires some extra code in the terrain system component. I think it's fine because we will register pass there anyway.